src/components/App.tsx

```diff
@@ -19,13 +22,37 @@ export default function App() {
     initAuth();
   }, [setCurrentUser]);
 
+  const [opened, { toggle }] = useDisclosure();
+
   return (
-    <div id="app-container">
-      <Header />
-      <Switch>
-        <Route path="/" component={Landing} />
-        <Route component={PageNotFound} />
-      </Switch>
-    </div>
+    <MantineProvider>
+      <AppShell
+        header={{ height: 60 }}
+        navbar={{
+          width: 300,
+          breakpoint: 'sm',
+          collapsed: { mobile: !opened },
+        }}
+      >
+        <AppShell.Header className={styles.header}>
+          <Burger opened={opened} onClick={toggle} hiddenFrom="sm" className={styles.burger} />
+          <Title className={styles.title}>Ribbit Recipes</Title>
+          <div className={styles.headerSpacer} />
+          <div className={styles.headerAuthContainer}>
+            <HeaderAuth />
+          </div>
+        </AppShell.Header>
+        <AppShell.Navbar className={styles.navbar}>Navbar</AppShell.Navbar>
```

Comment:
Maybe eventually the list of recipe books will live here? Or not, we could also drop the navbar entirely.